### ISSUE TITLE
Error when running Executive commands

### ISSUE THREAD

**Issue by JSProffitt** [tjdraper/executive-ee#2]
Received the following error when trying to run Executive with any command on a vanilla EE3.5.11 install:

```
<body>

<div id="content">

<h1>The following errors were encountered</h1>

<ul><li>This form has expired. Please refresh and try again.</li>
</ul>

<p><a href='JavaScript:history.go(-1)'>Return to Previous Page</a></p>

</div>

</body>
```

Security settings are as follows:

![image](https://user-images.githubusercontent.com/6521508/31521446-cb2883e8-af66-11e7-90e6-ad55d2475dbd.png)

**Comment by tjdraper** [comment]
I wonder if this has anything to do with MySQL strict mode. Perhaps because the settings column insert was not set properly the core_boot extension never got installed? Could you try uninstalling and re-installing as I cannot duplicate.

**Comment by tjdraper** [comment]
@JSProffitt Just checking in on this to see if you've had a chance to look at the above possible solution?

**Comment by JSProffitt** [comment]
Added new PR to add settings column to AddSessionStartHook migration query. Works as expected now.

**Comment by tjdraper** [comment]
Fixed in 1.0.1
